react/modules/withForwardedRef.js

```diff
@@ -1,13 +1,25 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 
-function withForwardedRef(Component) {
-  const ComponentWithRef = React.forwardRef((props, ref) => (
-    <Component {...props} forwardedRef={ref} />
-  ))
+// For more info see: https://stackoverflow.com/a/51127130/10725088
+const Element = typeof Element === 'undefined' ? function() {} : Element
+
+export const refShape = PropTypes.oneOfType([
```

Comment:
should this be defined here? shouldn't this be used across multiple components that are under similar circumstances?

Reply:
this is exactly why it's defined here. this way every component that need to add `ref to the `propTypes` can just import the shape to be used.

Reply:
ah sorry, my bad, I thought it was inside the `ActionMenu` component